test/Microsoft.Health.SqlServer.Tests.Integration/Features/Schema/SchemaInitializerTests.cs

```diff
@@ -0,0 +1,44 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Health.SqlServer.Features.Schema;
+using Xunit;
+using Xunit.Abstractions;
+
+namespace Microsoft.Health.SqlServer.Tests.Integration.Features.Schema
+{
+    public class SchemaInitializerTests : SqlIntegrationTestBase
+    {
+        public SchemaInitializerTests(ITestOutputHelper outputHelper)
+            : base(outputHelper)
+        {
+        }
+
+        [Fact]
+        public async Task DoesDatabaseExist_DoesNotExist_ReturnsFalse()
+        {
+            Assert.False(await SchemaInitializer.DoesDatabaseExistAsync(Connection, "doesnotexist", CancellationToken.None));
+        }
+
+        [Fact]
+        public async Task DoesDataBaseExist_Exists_ReturnsTrue()
```

Comment:
nit : Should we use Given_When_Then for readability. 
E.g. GivenDatabaseExists_WhenCreateDatabaseAsync_ThenReturnsTrue